.github/workflows/prod-release-deploy.yaml

```diff
@@ -40,3 +40,32 @@ jobs:
           apiKey: ${{ secrets.FLEEK_API_KEY }}
       - name: Get the output url
         run: echo "Deploy url is ${{ steps.deploy.outputs.deployUrl }}"
+      - name: Install Subgraph packages
+        working-directory: packages/subgraph
+        run: yarn install --frozen-lockfile
```

Comment:
Do you do the new install and build for every package because of `--frozen-lockfile`?
Just wondering because then we might not want to build and install everything from the root scripts as we do now since we basically install and build everything twice here.
What do you think is the best way to do it, @samepant?

Reply:
Whoops, I didn't realize the `preinstall` hook runs the installer for each package. That's handy. I'll adjust